ooniprobe/Utility/ThirdPartyServices.m

```diff
@@ -81,7 +81,14 @@ + (void)recordError:(NSString*)title reason:(NSString*)reason userInfo:(NSDictio
     NSError *error = [NSError errorWithDomain:title
                                          code:-1001
                                      userInfo:userInfo];
-    [SentrySDK captureError:error];
+    @try {
+        [SentrySDK captureError:error];
+    }@catch(NSException *e) {
+
+    }@finally {
+        NSLog(@"Failed reporting error");
```

Comment:
I wanted to point it out to you before i take a decision. The exception is as a result of an internal failure of Sentry to work with the data passed to it. We may need to update sentry or something. For now i just wanted to wrap it because it actually causes the app to crash.